ql/src/test/results/clientpositive/typechangetest.q.out

```diff
@@ -1258,10 +1258,10 @@ POSTHOOK: query: select cId, cTimeStamp from testAltColORC_n0 order by cId
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@testaltcolorc_n0
 #### A masked pattern was here ####
-1	2017-11-07 01:02:49.999999999
-2	1399-12-23 17:01:01.000000001
-3	1399-12-23 17:01:01.000000001
-4	1399-12-23 17:01:01.000000001
+1	2017-11-07 09:02:49.999999999
+2	1400-01-01 01:01:01.000000001
+3	1400-01-01 01:01:01.000000001
+4	1400-01-01 01:01:01.000000001
```

Comment:
Last time this was timezone sensitive to me, but if that's the case, it could be addressed in a followup.

Reply:
This one is expected. Behavior changed with new ORC version since timezone is being encoded in original files even if they do not contain a date/timestamp type columns (before we were not encoding this information unless those types were already present in the files). Then, when schema evolution kicks in, we use that tz value to interpret the values.